squape/squishserver.py

```diff
@@ -0,0 +1,52 @@
+import squish
+import test
+from remotesystem import RemoteSystem
+import os
+import pathlib
+from squape.report import log
+from squape.internal.exceptions import SquishserverError
+
+class SquishServer:
+    def __init__(self, location=None, host='127.0.0.1', port=4322):
+        if location is None:
+            self.location = os.environ['SQUISH_PREFIX']
+
+        self.host = host
+        self.port = port
+        self.remotesys = RemoteSystem(host, port)
+            
+
+    def addAUT(self, aut: str, path: str) -> None:
+        cmd = [
+            "squishserver", '--config', 'addAUT', aut, path
+        ]
+        cwd = self.location
+        log(f"Registering AUT {aut} with location: {path}")
+        (exitcode, stdout, stderr) = self.remotesys.execute(cmd, cwd)
+        if exitcode != "0":
+            raise SquishserverError("Squishserver was not able to register the AUT")
```

Comment:
Please add exitcode, stdout, and stderr to the Exception message, otherwise, it would be hard to investigate why the operation was not successful.